### PR TITLE
Streamlined run task reviews: Update API changelog, add integration docs

### PR DESCRIPTION
**Pull request by jarrodn-au**
### What

This PR adds details of streamlined run task reviews in Terraform Cloud, including integration documents, schemas, and an example payload.

Changed files: 
website/docs/cloud-docs/api-docs/changelog.mdx
website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx
website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx
website/public/schema/run-tasks/example-payload.json
website/public/schema/run-tasks/runtask-outcome.json
website/public/schema/run-tasks/runtask-result.json
website/public/schema/run-tasks/runtask-rich-label.json
website/public/schema/run-tasks/runtask-tag.json
website/public/schema/run-tasks/structured-run-task-result-schema.json

### Why

Explains how to use and integrate with the new streamlined run task reviews feature of Terraform Cloud.

PRD https://go.hashi.co/prd/tf-143
RFC https://go.hashi.co/rfc/tf-708

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [X] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [X] (N/A) Redirects have been added to `website/redirects.js` for moved, renamed, or deleted pages.
- [X] API documentation and the API Changelog have been updated. 
- [X] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [X] Pages with related content are updated and link to this content when appropriate.
- [X] (N/A)  Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [X] (N/A) New pages have metadata (page name and description) at the top.
- [X] (N/A) New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [X] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [X] UI elements (button names, page names, etc.) are bolded.
- [X] The Vercel website preview successfully deployed.

#### Reviews
- [X] I or someone else reviewed the content for technical accuracy.
- [X] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx [hunk 1]
```diff
@@ -157,12 +167,53 @@ Status values other than passed, failed, or running return an error. Both the pa
...
+| `body`     | string | (Optional)  A detailed message for the result in Markdown format. |
+| `url`     | string | (Optional) A URL that a user may browse to for more information about this result. |
+| `tags`     | object | (Optional) An object containing tag arrays, named by the property key. |
+| `tags.key`     | string |The name of the header tag. This should be a very short name ideally no more than two or three words. Special handling is given to `severity` and `status` keys. |
+| `tags.key[].label`     | string | The text value of the tag. |
+| `tags.key[].level`     | enum string | (Optional) The error level for the tag. Defaults to `none`. Either `none`, `info`, `warning`, or `error`. For levels other than `none`, labels are rendered with color and icon for that level. |
+
+##### Severity and Status Tags
+
+When surfacing run task outcomes with tags "severity", or "status", those tags will be used to enrich the list display of outcomes in Terraform Cloud, enabling earlier feedback of severity and statuses.
```
Comment (rkoron007) [inline website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx:198]:
```suggestion
Run task outcomes with tags named "severity" or "status" are enriched within the outcomes display list in Terraform Cloud, enabling an earlier response to issues with severity and status.
```

File: website/docs/cloud-docs/integrations/run-tasks/index.mdx [hunk 2]
```diff
@@ -60,27 +63,31 @@ Refer to the [Run Task Integration API](/terraform/cloud-docs/api-docs/run-tasks
...
 
-Terraform Cloud expects this callback within 10 minutes, or the task will be considered to have `errored`. The supplied message attribute will be displayed in Terraform Cloud on the run details page. The status can be `passed` or `failed`.
+Terraform Cloud expects this callback within 10 minutes, or the task will be considered to have `errored`. The supplied message attribute will be displayed in Terraform Cloud on the run details page. The status can be `running`, `passed` or `failed`.
 
 Here's what the data flow looks like:
 
 ![Screenshot: a diagram of the user and data flow for a Terraform Cloud run task](/img/docs/terraform-cloud-run-tasks-diagram.png)
 
+
+Refer to the [Run Task Integration API](/terraform/cloud-docs/api-docs/run-tasks/run-tasks-integration#structured-results) for the exact payload specification, as well as the [Run Task JSON Schema](/schema/run-tasks/runtask-result.json) for code generation and payload validation.
```
Comment (rkoron007) [inline website/docs/cloud-docs/integrations/run-tasks/index.mdx:88]:
```suggestion
Refer to the [run task integration API](/terraform/cloud-docs/api-docs/run-tasks/run-tasks-integration#structured-results) for the exact payload specifications, and the [run task JSON schema](https://github.com/hashicorp/terraform-docs-common/blob/main/website/public/schema/run-tasks/runtask-result.json) for code generation and payload validation.
```

**Review by rkoron007 — APPROVED** [review]
Looks good! Mostly just small edits for style, but please feel free to push back on anything that doesn't feel good. ✨

**Review by hcrhall — APPROVED** [review]
LGTM!

**Pull request merged**